var/spack/repos/builtin/packages/nco/package.py

```diff
@@ -62,8 +84,10 @@ class Nco(AutotoolsPackage):
     conflicts("%gcc@9:", when="@:4.7.8")
 
     def configure_args(self):
-        spec = self.spec
-        return [f"--{'enable' if '+doc' in spec else 'disable'}-doc"]
+        return [
+            f"{self.enable_or_disable('+doc')[0]}-doc",
+            f"{self.enable_or_disable('+openmp')[0]}-openmp",
+        ]
```

Comment:
This one doesn't seem correct, you can have a look at other packages using `enable_or_disable` to fix the call here. It should be something like:
```suggestion
        config_args = self.enable_or_disable("doc")
        config_args += self.enable_or_disable("openmp")
        return config_args
```

Reply:
Thanks @alalazo.  @altheaden, could you please make those changes?  I will test locally to make sure they work.

Reply:
Everything worked out fine in my local test with this change.

Reply:
I guess I need to revisit what the original output was expected to be.

Reply:
🤦